tests/test_move.py

```diff
@@ -6,10 +6,13 @@
 
 import fs.move
 from fs import open_fs
+from fs.errors import ResourceReadOnly
+from fs.path import join
+from fs.wrap import read_only
 
 
 @parameterized_class(("preserve_time",), [(True,), (False,)])
-class TestMove(unittest.TestCase):
+class TestMovePreserveTime(unittest.TestCase):
```

Comment:
Perhaps this should be called `TestMoveCheckTime`, to reflect the fact that the times _aren't_ always preserved? :shrug:

Reply:
Yes, that's better!